bundles/net.tourbook/src/net/tourbook/ui/views/tourDataEditor/TourDataEditorView.java

```diff
@@ -4106,7 +4106,8 @@ private void createUI_Section_147_Weather_Other_Col1(final Composite parent) {
          // spinner: pressure value
          _spinWeather_PressureValue = new Spinner(container, SWT.BORDER);
          _spinWeather_PressureValue.setToolTipText(Messages.Tour_Editor_Label_AirPressure_Tooltip);
-         _spinWeather_PressureValue.setMaximum(400000);
+         //The highest barometric pressure ever recorded on Earth was 32.01 inches (1083.98), measured in Agata, U.S.S.R., on December 31, 1968.
+         _spinWeather_PressureValue.setMaximum(108398);
```

Comment:
When there would be a new record, it could not be entered into MT

Reply:
Sorry, I don't understand what you mean. No data cannot be entered ? why ?

Reply:
You have set the max value for air pressure to 108398 but when there would be a higher value than 108398 (a new record), it could not be entered in this field

Reply:
Oh, Ok. I will increase the max value.